doc/ecs/components/animation_component.md

```diff
@@ -3,18 +3,53 @@ title: "Animation Component"
 ---
 
 ## Wofür
-- Wofür braucht es diese Comonent?
-- Was macht es? 
+
+Das `AnimationComponent` ist dafür zuständig, die Animationen einer Entität zu verwalten. Es arbeitet mit dem `DrawSystem`
+zusammen, welches im Endeffekt dafür sorgt, dass irgendetwas auf dem Bildschirm angezeigt wird. Es gibt immer eine aktuelle
+Animation, die von einer Entität abgespielt wird. Diese Animation kann während der Laufzeit durch die entsprechenden Methodenaufrufe
+geändert werden.
 
 ## Aufbau
 
-- UML
-- Aufbau Erklären
-- Erklären warum Animationen auch in anderen Components gespeichert werden
+![UML Diagram](../img/animationComponent.png)
```

Comment:
dito: ist das Bild nur hier relevant? dann bitte in einen `img/`-ordner auf dieser ebene verschieben und den link anpassen.